.github/workflows/cp-deployment-branch.yml

```diff
@@ -68,6 +68,62 @@ jobs:
         env:
           GITHUB_TAG: ${{ github.sha }}
 
+      - name: Notify Slack (success only)
+        if: ${{ success() }}
+        uses: slackapi/slack-github-action@v1
+        with:
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": "Deployment Successful! :partying_face:",
+                    "emoji": true
+                  }
+                },
+                {
+                  "type": "section",
+                  "fields": [
+                    { "type": "mrkdwn", "text": "*Project:* ${{ github.event.repository.name }}" },
+                    { "type": "mrkdwn", "text": "*When:* ${{ github.run_started_at }}" },
+                    { "type": "mrkdwn", "text": "*Branch:* \n${{ github.ref_name }}" },
+                    { "type": "mrkdwn", "text": "*Successful Job:* \n${{ github.job }}" }
+                  ]
+                },
+                {
+                  "type": "actions",
+                  "elements": [
+                    {
+                      "type": "button",
+                      "action_id": "deployment_link",
+                      "text": { "type": "plain_text", "text": "View Run" },
+                      "url": "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+                    }
+                  ]
+                }
+              ]
+            }
+        env:
+            SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+      - name: Notify Slack on failure (main branch only)
+        if: ${{ failure() && github.ref_name == 'main' }}
```

Comment:
This workflow is for branch deployments and referencing main is not appropriate here.